docs/user-guide/installation/index.rst

```diff
@@ -42,6 +42,10 @@ Prerequisites
   `serious limitations <http://docs.mongodb.org/manual/faq/fundamentals/#what-are-the-32-bit-limitations>`_
   on 32-bit architectures. It is strongly advised that you run MongoDB on a 64-bit architecture.
 
+.. note::
+  `MongoDB documentation <https://docs.mongodb.com/manual/tutorial/transparent-huge-pages/>`_
+  suggests disabling Transparent Huge Pages to ensure best performance with MongoDB.
```

Comment:
What about putting the link on the text `disabling Transparent Huge Pages` instead of `MongoDB documentation`?